spec/rubocop/cop/rspec/verified_doubles_spec.rb

```diff
@@ -74,6 +74,14 @@
     RUBY
   end
 
+  it 'ignores doubles that have no name but methods specified' do
```

Comment:
Could you add a similar test in the block above (line 56–67) that we detect an offense on the same code when `IgnoreNameless` is false?

Reply:
@bquorning done